bookish/Services/return_services.py

```diff
@@ -0,0 +1,87 @@
+from bookish.models import db
+from bookish.models.users import Users
+from bookish.models.books import Books
+from bookish.models.checked_out_books import Checked_Out_Books
+from bookish.Services.auxiliary_functions import check_if_authenticated, return_book_by_ISBN, return_user_by_user_id
+from bookish.error_handler import *
+
+
+def list_all_users():
+    try:
+        all_users = Users.query.all()
+    except Exception as e:
+        raise Exception({"status": "Error", "error_message": str(e)})
+
+    return all_users
+
+
+def list_all_books(parsed_request):
+    try:
+        all_books = Books.query.all()
+    except Exception as e:
+        raise Exception({"status": "Error", "error_message": str(e)})
+    all_books = apply_list_filters(all_books,  order, limit)
+    return all_books
+
+
+def list_books_by_title(auth_token, book_title):
+    try:
+        all_books_by_title = filter(lambda book: book.book_title==parsed_request['book_title'],Books.query.all())
+    except Exception as e:
+        raise Exception({"status": "Error", "error_message": str(e)})
+    books_by_title = apply_list_filters(all_books_by_title, parsed_request['order'], parsed_request['limit'])
+    return books_by_title
+
+
+def list_books_by_author(auth_token, parsed_request : dict):
+    try:
+        print("here")
+        all_books_by_author = filter(lambda book: book.book_author==parsed_request['book_author'],Books.query.all())
+    except Exception as e:
+        raise Exception({"status": "Error", "error_message": str(e)})
+
+    books_by_author = apply_list_filters(all_books_by_author, parsed_request['order'], parsed_request['limit'])
+    return books_by_author
+
+def apply_list_filters(books, order, limit):
+    if order == 'ASC':
+        books = sorted(books, key=lambda book: book.book_title)
+    else:
+        books = sorted(books, key=lambda book: book.book_title, reverse=True)
+    if limit == -1:
```

Comment:
I think we can do something cleaner instead of hardcoding `-1` to indicate there is no limit.